hazelcast/src/main/java/com/hazelcast/config/XmlConfigBuilder.java

```diff
@@ -500,6 +480,46 @@ private void handleWanReplication(final Node node) throws Exception {
         config.addWanReplicationConfig(wanReplicationConfig);
     }
 
+    private void handleWanTargetConfig(WanReplicationConfig wanReplicationConfig,
+                                       WanTargetClusterConfig wanTarget, Node targetChild) {
+        final String targetChildName = cleanNodeName(targetChild);
+        if ("replication-impl".equals(targetChildName)) {
```

Comment:
Node start will fail with `ClassNotFoundException` in that case. But I think this part is out of scope of this since there is nothing new here. I just extracted them to a method logic is same as before.